plugins/inputs/postgresql/postgresql.go

```diff
@@ -53,6 +54,11 @@ var sampleConfig = `
   ## A list of databases to pull metrics about. If not specified, metrics for all
   ## databases are gathered.  Do NOT use with the 'ignored_databases' option.
   # databases = ["app_production", "testing"]
+
+  ## Whether to use prepared statements when connecting to the database.
+  ## This should be set to false when connecting through a PgBouncer instance
+  ## with pool_mode set to transaction.
+  prepared_statements = true
```

Comment:
Can we make this a default and comment the option here?
```suggestion
  # prepared_statements = true
```